lib/ipython_autoimport.py

```diff
@@ -214,16 +235,39 @@ def _uninstall_namespace(ipython):
     ipython.user_ns = ipython.Completer.namespace = dict(ipython.user_ns)
 
 
+@magic_arguments()
+@argument("-c", "--clear", type=str, help="Clear cache for this symbol")
+@argument("-h", "--history", dest="history", action="store_const",
+            const=True, default=False, help="Show autoimports from this session")
+def autoimport(arg):
+    ipython = get_ipython()
+    args = parse_argstring(autoimport, arg)
+
+    if args.clear:
+        if ipython.user_ns._import_cache.pop(args.clear, None):
+            _report(ipython,
+                    "cleared symbol '{}' from autoimport cache.".format(args.clear))
+        else:
+            _report(ipython, 
+                    "didn't find symbol '{}' in autoimport cache.".format(args.clear))
+
+    if args.history:
+        if ipython.user_ns._imported:
+            _report(ipython, 
+                    "the following were autoimported:\n{}".format(
```

Comment:
perhaps "the following autoimports were run" ("the following what were autoimported"?)